src/raiden_libs/matrix.py

```diff
@@ -56,12 +57,12 @@ def __init__(
 
     def _run(self):
         self.client.start_listener_thread()
-        self.client.sync_thread.get().wait()
```

Comment:
Are you sure about this? I am not sure about the effect of this

Reply:
Ehm it works like that. But I don't really know what it does either. I will put it on the open questions list